ui-tests/tools/com.liferay.ide.ui.workspace/com.liferay.ide.ui.workspace.wizard/com.liferay.ide.ui.workspace.wizard.validation.tomcat.70/src/com/liferay/ide/ui/workspace/tests/ValidationLiferayWorkspaceTests.java

```diff
@@ -60,6 +61,19 @@ public void checkBuildType() {
 	public void checkBundleUrl() {
 	}
 
+	@Test
+	public void checkEnableTargePlatform() {
```

Comment:
in this case, when select enable target platform, we should check the following:
1. Target platform state(can be select)
2. Target platform default state(according to liferay version)
3. all available target platform values
4. index source state(can be edit)